app/services/notifications/submit-returned-letter.service.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+/**
+ * Orchestrates updating the returnedAt date for a letter
+ *
+ * @module SubmitReturnedLetterService
+ */
+
+const NotificationModel = require('../../models/notification.model.js')
+
+/**
+ * Orchestrates updating the returnedAt date for a letter
+ *
+ * @param {string} notifyId - The id of the notification
+ *
+ * @returns {Promise<object>} - The returned database object
+ */
+async function go(notifyId) {
+  await NotificationModel.query().patch({ returnedAt: new Date() }).where('notifyId', notifyId)
```

Comment:
We need to know if we receive a call back but cannot find the matching letter. We just need to log it, but it means we need to wrap this in something that checks we did update a record.

I'd suggest using `.returning('id')` (see [returning tricks](https://vincit.github.io/objection.js/recipes/returning-tricks.html#postgresql-returning-tricks)) and then checking if you actually have an ID.

If not, log (`omg` vs `omfg` - no need for an error) the fact we failed to find a matching notification for the callback.